test/functionalTest/cases/1068_metadata_value_as_compound/metadata_value_as_compound_array.test

```diff
@@ -0,0 +1,106 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Some test notifying compound metadata should be included, to check that they are correctly rendered in notification payloads (current test only consider queries, as fas as I have checked).